src/libs/Notification/PushNotification/subscribeToPushNotifications.ts

```diff
@@ -142,7 +145,7 @@ function navigateToReport({reportID, reportActionID}: ReportActionPushNotificati
                     Navigation.goBack();
                 }
 
-                Log.info('[PushNotification] onSelected() - Navigation is ready. Navigating...', false, {reportID, reportActionID});
```

Comment:
NAB: You're sure the reportActionID isn't useful in the logs?

Reply:
Yeah since we're just navigating to the report, it's not really important here.